db/migrate/20160808150745_add_metrics_status_to_ext_management_system.rb

```diff
@@ -0,0 +1,7 @@
+class AddMetricsStatusToExtManagementSystem < ActiveRecord::Migration[5.0]
+  def change
+    add_column :ext_management_systems, :last_metrics_error, :text
+    add_column :ext_management_systems, :last_metrics_update_date, :timestamp
+    add_column :ext_management_systems, :last_metrics_success_date, :timestamp
```

Comment:
@simon3z @moolitayer @blomquisg Update:
`last_metrics_error` column now only holds the error, or `nil` to indicate a valid status.
This is consistent with the way we use `last_refresh_error`